src/NewPlasterManifest.ps1

```diff
@@ -0,0 +1,170 @@
+    in and below the current directory (except for any plasterManifest files).
+.INPUTS
+    None
+
+    You cannot pipe input to this cmdlet.
+.OUTPUTS
+    None
+.LINK
+    Invoke-Plaster
+#>
+function New-PlasterManifest {
+    [CmdletBinding(SupportsShouldProcess=$true)]
+    param(
+        # Specifies the path and file name of the new Plaster manifest. Enter a path and file name with a .xml file name
+        # extension, such as $pshome\Modules\MyPlasterTemplate\plasterManifest.xml.  NOTE: Plaster requires the manifest
+        # file be named either plasterManifest.xml OR plasterManifest_<culture-name>.xml e.g. plasterManifest_fr-FR.xml.
+        # The default, if no value is provided is to create plasterManifest.xml in the current directory.
+        [Parameter(Position=0)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Path = "$pwd\plasterManifest.xml",
+
+        # Unique identifier for all versions of this template.  The id is a GUID.  Use the same id for each version
+        # of your template.  This will prevent editor environments from listing multiple, installed versions of your
+        # template. When the id stays the same, the editor will list only the latest version of your template.
+        [Parameter()]
+        [Guid]
+        $Id = [guid]::NewGuid(),
+
+        # Specifies the version of the template.
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [ValidatePattern('^\d+\.\d+(\.\d+((\.\d+|(\+|-).*)?)?)?$')]
+        [string]
+        $TemplateVersion = "1.0.0",
+
+        # Title of the Plaster template.  This short string is typically used in an editor like VSCode when displaying
+        # a list of Plaster templates.  A typical title might be "New DSC Resource" or "New PowerShell Module".
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Title,
+
+        # Description of the Plaster template.  This is a longer descrition of what the the template is for. It
+        # is typically used in an editor like VSCode when displaying additional information about a Plaster template.
+        # A typical title might be "Creates files required for a PowerShell module with optional support for Pester
+        # tests, building with psake and publishing to the PowerShell Gallery."
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Description,
+
+        # Specifies an array of tags for the template.  Users can search for templates based on these tags.
+        [Parameter()]
+        [string[]]
+        $Tags,
+
+        # Specifies the author of the template.
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Author,
+
+        # If specified, the contents of the directory the manifest is being created in will be added to the
+        # manifest's content element.
+        [Parameter()]
+        [switch]
+        $AddContent
+    )
+
+    begin {
```

Comment:
Habit at this point because `begin` runs just once.  But, given there is no `process` block and `end` runs just once, that variable could just as well have been defined in the `end` block.  OTOH if someone where to ever add a `process` block then the variable would be defined in the correct spot.